README.rst

```diff
@@ -45,6 +45,17 @@ elections server::
 Once that is running, simply go to http://localhost:5002/ in your browser on
 your host to see your running elections test instance.
 
+A note about Fonts
```

Comment:
Nitpick: maybe lowercase the F in Fonts so it's consistent with the style of the rest of this heading?